users/templates/base.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap" rel="stylesheet">
+    
+    <script src="https://cdn.jsdelivr.net/npm/sweetalert2@10"></script>
+    
+    {% load static %}
+    <link href="{% static 'base.css' %}" rel="stylesheet">
+
+    {% block head-content %}
+    {% endblock %}
+        
+  </head>
+	<body>
+    <nav role="navigation">
+      <div id="menuToggle">
+        <input type="checkbox" />
+        <span></span>
+        <span></span>
+        <span></span>
+        <ul id="menu">
+          <a href="{% url 'welcome_page' %}"><li><i class="fa fa-home" aria-hidden="true" style="font-size:24px;"></i>&nbsp;&nbsp; Home</li></a>
+          <a href="#"><li><i class="fa fa-id-card" aria-hidden="true" style="font-size:24px;"></i>&nbsp; Contact</li></a>
+          <a href="/admin"><li><i class="fa fa-user-secret" aria-hidden="true" style="font-size:24px;"></i>&nbsp; Admin</li></a>
```

Comment:
Do we really need the Admin as part of the menu?

Reply:
this is for tests purposes only, 
Because we relay on the admin in the development of the site
this is an easier way to go to the admin site.

We will remove it in a later stage.